packages/bruno-cli/src/runner/run-single-request.js

```diff
@@ -186,6 +195,12 @@ const runSingleRequest = async function (
       request.data = qs.stringify(request.data);
     }
 
+    if (request.headers['content-type'] === 'multipart/form-data' && !request.__bruno__bodySetViaMethodCall) {
```

Comment:
Instead of checking of the Body was set inside a script. Wouldn't it be better to check if the body is already a `FormData` Object?

What if a user has a script like this:
```
const originalBody = req.getBody();

const calculated = 'Imagine this is calculated';

req.setBody({
  ...originalBody,
  'script-val': calculated,
})
```
Now the plain object is sent to the server.